src/test/java/showcase/neodymium/tests/slider/SliderTest.java

```diff
@@ -0,0 +1,96 @@
+package showcase.neodymium.tests.slider;
+
+import static com.codeborne.selenide.Selenide.$;
+
+import org.junit.Test;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Selenide;
+import com.xceptance.neodymium.util.SelenideAddons;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.Owner;
+import io.qameta.allure.Severity;
+import io.qameta.allure.SeverityLevel;
+import io.qameta.allure.junit4.DisplayName;
+import io.qameta.allure.junit4.Tag;
+import showcase.neodymium.tests.AbstractTest;
+
+/**
+ * @author j.sengbusch
+ */
+@Severity(SeverityLevel.TRIVIAL)
+@Owner("Johannes Sengbusch")
+@Tag("smoke")
+@DisplayName("Slidertest")
+public class SliderTest extends AbstractTest
+{
+
+    @Test
+    @Description(value = "Showcase for a horizontal drag and drop of a web slider to the right side.")
+    public void testHorizontalRightMovement()
+    {
+        // open demo page
+        Selenide.open("https://demos.telerik.com/kendo-ui/slider/index");
+
+        // move the slider to the right
+        // offset for the horizontal movement: 40
+        // offset for the vertical movement: 0
+        // until the value: 8
+        // retries: 5
+        // pause between movements: 3000ms
```

Comment:
Order the comments as the method expects them to come. This makes things more clear. Also fix it for the rest please.